packages/ui/src/components/Command/AiCommand.tsx

```diff
@@ -412,20 +514,62 @@ const AiCommand = () => {
                         )}
                       </>
                     </div>
+
+                    {message.status === MessageStatus.Complete &&
+                      message.supportFirstPrompt === true &&
+                      aiVariant === 'support' && (
+                        <div className="flex items-center justify-center mt-4 space-x-2">
```

Comment:
There seems to be a spacing issue between this and the next item:

<img width="777" alt="image" src="https://github.com/supabase/supabase/assets/26616127/a78adb07-5a3f-4986-9c2e-548d0043076a">
<img width="773" alt="image" src="https://github.com/supabase/supabase/assets/26616127/7386252e-df81-414c-969c-25a37a6740cd">

Reply:
Ah yes, I'm aware of this one but it's still on my list to fix for this week.